Programming_in_Scala/Chap5_Types_Operations/Readme.md

```diff
@@ -0,0 +1,39 @@
+Basic Types and Operations
+
+->  Every value in scala is an object and every operation on an object is a method call
+
+->  scala doesnot have operators . All the operations are method calls
+
+->  1+2 in scala is calling the method '+' in the integer value 1 which takes one argument ie 2 . so in effect it is 1.+(2)
+
+->  All the members of package scala and java.lang are automatically imported into scala sourec file
```

Comment:
typo source . Scala  contains Int, float etc and java.lang contains string . mention about pre-def as well